string name
float32 attention_value
float32 boost_value
string[] actions
string[] required_resources
